src/main/java/uk/ac/ebi/ena/webin/cli/manifest/ManifestFileSuffix.java

```diff
@@ -20,9 +20,6 @@ public interface ManifestFileSuffix {
 
   List<String> GZIP_OR_BZIP_FILE_SUFFIX = Arrays.asList(GZIP_FILE_SUFFIX, BZIP2_FILE_SUFFIX);
 
-  List<String> AGP_FILE_SUFFIX =
-      Arrays.asList(".agp" + GZIP_FILE_SUFFIX, ".agp" + BZIP2_FILE_SUFFIX);
```

Comment:
is AGP support getting completely removed from webin-cli? won't other submission contexts need it?

Reply:
Yes Haseeb its getting removed completely.